plerkle_messenger/src/redis_messenger.rs

```diff
@@ -136,6 +135,17 @@ impl RedisMessenger {
             };
 
             if info.times_delivered > self.retries {
+                // --------------------------------------------------------------------------------
+                let extracted_id = extract_id(bytes, stream_key);
+                match extracted_id {
+                    Ok(id) => {
+                        log!(target: "inspect_dropped_msg", Level::Warn, "{} ID: {}", stream_key, id,);
```

Comment:
why did you use `log` instead of `warn!`? Actually can you please use `error` here instead, since its the highest log level so we'll always catch it. I believe account ingester only logs errors to reduce cost rn.